sys-kernel/raspberrypi-image/raspberrypi-image-4.14.98_p20190215.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot
+
+DESCRIPTION="Raspberry Pi (all versions) kernel and modules"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+LICENSE="GPL-2 raspberrypi-videocore-bin"
+SLOT="0"
+RESTRICT="binchecks strip"
```

Comment:
Why strip? It's allowed and other devs also said on my other packges, that strip doesn't count as "do not modify". I also would drop binchecks, as no QA errors show up.